Cargo.lock

```diff
@@ -3642,6 +3671,28 @@ version = "0.1.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "e9426b2a0c03e6cc2ea8dbc0168dbbf943f88755e409fb91bcb8f6a268305f4a"
 
+[[package]]
```

Comment:
looks like you need to run `cargo update` to get rid of this unused dep

Reply:
[549075e](https://github.com/n0-computer/iroh/pull/1241/commits/549075e5a679827246f00a3b24a45fab16098448) should do it